### PR TITLE
Bugfix/atr 202 master

### PR DESCRIPTION
**Pull request by SENya1990**

### CODE REVIEW

**Comment by SENya1990** [comment]
@dropsonic I have changed the folder structure for PX1021, PX1023 and PX1014 diagnostics and tests as we discussed. Please take a look

File: src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/DAC/DacFields/PX1014/NonNullableTypeForBqlFieldTests.cs [hunk 1]
```diff
@@ -17,46 +17,30 @@ namespace Acuminator.Tests
...
-						new DiagnosticResultLocation("Test0.cs", line, column)
-					}
-			};
-
-		    return diagnostic;
-	    }
-
 	    [Theory]
-	    [EmbeddedFileData(@"Dac\PX1014\Diagnostics\NonNullableTypeForBqlField.cs")] 
+	    [EmbeddedFileData(new[] { @"Diagnostics\NonNullableTypeForBqlField.cs" }, true)]
```
Comment (dropsonic) [inline src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/DAC/DacFields/PX1014/NonNullableTypeForBqlFieldTests.cs]: This is not convenient; consider creating several constructors with a different number of arguments.
But again, folder structure should be discussed first.

Comment (SENya1990) [reply src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/DAC/DacFields/PX1014/NonNullableTypeForBqlFieldTests.cs]: @dropsonic I agree that we need to discuss it in details, this is a prototype, based on existing  proposals - to group together analyzer and fix, sources and code.

**Comment by SENya1990** [comment]
Will be reimplemented without changes in folder structure

**Pull request closed without merging**